app/src/main/java/com/omkarmoghe/pokemap/app/App.java

```diff
@@ -0,0 +1,32 @@
+package com.omkarmoghe.pokemap.app;
+
+import android.app.Application;
+
+import com.omkarmoghe.pokemap.dagger.AppModule;
+import com.omkarmoghe.pokemap.dagger.DaggerMainComponent;
+import com.omkarmoghe.pokemap.dagger.MainActivityModule;
+import com.omkarmoghe.pokemap.dagger.MainComponent;
+
+/**
+ * Custom application class.
+ * <p>
+ * Created by fess on 21.07.16.
+ */
+public class App extends Application {
+
+    private MainComponent mainComponent;
```

Comment:
This should be a static field so that accessors don't have to cast getActivity() when injecting.